<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findDlg</class>
 <widget class="QDialog" name="findDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="findLine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="findLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxGroups">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>74</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Groups</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxChildren">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>74</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Children</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxInclSiblings">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Include siblings in results</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxExactMatch">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exact Match</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DuplicatesBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplicates</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>findDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>findDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
